henriqueApp/app/users.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { View, Text, FlatList, StyleSheet } from 'react-native';
+
+type User = {
+  id: number;
+  name: string;
+  email: string;
+};
```

Comment:
```suggestion
interface User {
  id: number;
  name: string;
  email: string;
};
```

Preferimos a convenção de usar interfaces, porque elas podem ser estendidas.